_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Beginner Tutorial'
+excerpt: In this tutorial we'll walk through the basics of KittyCAD and using the API
+langs: ['py']
+---
+
+Welcome to KittyCAD. We are building tools to be the backbone infrastructure for hardware tools. 
+
+Welcome to our beginner tutorial. This is primarily for those that have little coding experience, have no idea what an API is, or have no idea how to use them. 
+
+## What is an API?
+
+API stands for Application Programming Interface. The key to an APi is that it allows applications to talk to each other. Here are some current examples of API's used today.
+* Uber drivers have Google Maps built into their Uber app for naviation.
+* Duolingo allows users to log in using their Facebook information.
+* Many applications use Stripe for processing payments. 
+
+API's are powerful. Let's take the Uber example above. It would take a lot of time and engineers to develop a map feature like similar Google Maps within the Uber app. Uber uses Google Maps' API to access it within their own app. Of course, Uber pays a portion of their sales to Google in return for using their API. 
+
+## Let's Get Started
+
+For you to communicate with KittyCAD's website, you must have an API key. When you make an API command, you will pass this APi key along with your command, so that we can verify it's you. 
+
+If you have already generated an API key, you will see it shown below. If you see "No API keys have been generated yet". In order for you to see your API key, you must do the following:
+* Create an account. If you haven't already, you can [create an account here](/account)
+* Sign in. Once you've signed up for an account, make sure you are signed in. 
+* Generate an API key. You can either go into your [account settings](/account?tab=api_tokens) to generate an API key, or you can press the button below and we will take care of it for you!
+
+Generate API Key button here
```

Comment:
https://github.com/KittyCAD/website/pull/882 has been merged, so you should be okay add `<MarkDownAPITokens />`